libs/core/include_local/include/hpx/local/numeric.hpp

```diff
@@ -10,5 +10,5 @@
 #include <hpx/parallel/numeric.hpp>
 
 namespace hpx {
-    using hpx::parallel::adjacent_difference;
+
 }    // namespace hpx
```

Comment:
You can remove the whole `namespace hpx {}` here.

Reply:
I removed whole "namespace hpx {}"